README.md

```diff
@@ -81,6 +81,21 @@ vars:
     quickbooks_union_databases: ['quickbooks_usa','quickbooks_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ```
 
+### Customize the ordinal for account classes in `dbt_quickbooks` models 
+The current default numbering for ordinals is based on best practices for balance sheets and profit-and-loss statements in accounting. You can see these ordinals in action in the `quickbooks__general_ledger_by_period`, `quickbooks__balance_sheet` and `quickbooks__profit_and_loss` models. The ordinals are assigned off of the `account_class` values.
+ 
+If you'd like to modify this, take the following steps:
+
+1) Import a csv with fields into the `seeds` folder, then configure the `financial_statement_ordinal` variable in your `dbt_project.yml` to reference the seed file name. 
+2) Examine the `financial_statement_ordinal_example` file in the `integration_tests/seeds` folder to see what your sample seed file should look like. (NOTE: Make sure that your `seed` file name is different from `financial_statement_ordinal_example` to avoid errors.). You can use this file as an example and follow the steps in (1) to see what the ordering of the data looks like. 
+3) When adding and making changes to the seed file, you will need to run the `dbt build` command to compile the updated seed data into the above financial reporting models.
+
+These are our recommended best practices to follow with your seed file (you can see them in action in the `financial_statement_ordinal_example` file): 
+- REQUIRED: In each row of the seed file, only populate ONE of the columns of `account_class`, `account_type`, `account_sub_type`, and `account_number` to avoid duplicated ordinals and test failures. This should also make the logic cleaner in defining which account value takes precedence in the ordering hierarchy. 
```

Comment:
```suggestion
- REQUIRED: In each row of the seed file, only populate **ONE** of the `account_class`, `account_type`, `account_sub_type`, and `account_number` columns to avoid duplicated ordinals and test failures. This should also make the logic cleaner in defining which account value takes precedence in the ordering hierarchy. 
```